incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/airline/ConfigGenerator.java

```diff
@@ -58,7 +60,18 @@ public abstract class ConfigGenerator
 
     public abstract String generate();
 
-    protected String resolveContentType()
+    protected boolean jsonContentType()
```

Comment:
As a method name, makes sense to use `hasJsonContentType()` here.

Reply:
OK